config_files/tum_odometry.yaml

```diff
@@ -1,6 +1,6 @@
 %YAML:1.0
 camera_calibration_file: "/home/sarmento/rgbd_rtk/config_files/kinect_default.yaml"
-index_file: "/home/sarmento/Downloads/Dataset/tgz/rgbd_dataset_freiburg2_desk/index.txt"
+index_file: "/home/hmarcos/Downloads/TUM-Datasets/rgbd_dataset_freiburg1_rpy/index"
```

Comment:
Please, change this file with the following:
camera_calibration_file: "/path/to/camera_calibration.yaml"
index_file: "/path/to/dataset/dataset_specific/index"

In this way, the dataset loader (e.g. RGBDLoader) will exit the program with a message informing the user that the requested dataset couldn't be loader.

This forces the user to change these paths according to its local configuration.

Reply:
Solved on commit 2bfa90aeef83a551865b3157a2cb7f11f79b5774